doc/decision-tree-widget/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "decision-tree-widget",
+  "version": "1.0.0",
+  "description": "Interactive widget for a decision tree to assist choosing an RxJS operator",
+  "scripts": {
+    "yaml2json": "rm -f src/tree.json && node tools/yaml2json tree.yml ./src/tree.json",
+    "prebrowserify": "mkdir -p dist",
+    "browserify": "browserify src/main.js -t babelify --outfile dist/decision-tree-widget.js",
+    "build": "npm install && npm run yaml2json && npm run browserify",
+    "postbuild": "rm -f src/tree.json"
+  },
+  "dependencies": {
+    "@motorcycle/core": "^1.1.0",
+    "@motorcycle/dom": "^1.2.1",
+    "most": "^0.18.1",
```

Comment:
This is so good, it's nearly up there with when that gentleman asked @blesh on his article, if he'd used RxJS!!
#acidental-trolling  :trollface: